xmtp_mls/migrations/2023-10-29-205333_state_machine_init/up.sql

```diff
@@ -5,8 +5,8 @@ CREATE TABLE groups (
     "created_at_ns" BIGINT NOT NULL,
     -- Enum of GROUP_MEMBERSHIP_STATE
     "membership_state" INT NOT NULL,
-    -- Last time the installation list was checked for the purpose of seeing if any are missing
-    "installation_list_last_checked" BIGINT NOT NULL
+    -- Last time the installations were checked for the purpose of seeing if any are missing
+    "installations_last_checked" BIGINT NOT NULL
```

Comment:
I think one consequence of being in "alpha" (even with our many warnings of breaking changes) is that we should start adding new migrations rather than updating old ones.

Reply:
updated to use a new migration in https://github.com/xmtp/libxmtp/pull/537/commits/c13abe42cee0d88d5dfca1511d40f61b8f36bb9e

Reply:
True true. + The buidl week has officially begun.  No need to break early adopters if we can avoid it.